lxd/instance/drivers/driver_qemu.go

```diff
@@ -7061,6 +7062,50 @@ func (d *qemu) checkFeatures(hostArch int, qemuPath string) ([]string, error) {
 		features = append(features, "cpu_hotplug")
 	}
 
+	// Check AMD SEV features (only for x86 architecture)
+	if hostArch == osarch.ARCH_64BIT_INTEL_X86 {
+		cmdline, err := os.ReadFile("/proc/cmdline")
+		if err != nil {
+			return nil, err
+		}
+
+		parts := strings.Split(string(cmdline), " ")
+
+		// Check if SME is enabled in the kernel command line.
+		if slices.IndexFunc(parts, func(p string) bool { return p == "mem_encrypt=on" }) != -1 {
+			features = append(features, "sme")
+		}
+
+		// Check if SEV/SEV-ES are enabled
+		sev, err := os.ReadFile("/sys/module/kvm_amd/parameters/sev")
+		if err != nil {
+			logger.Debug("Failed reading SEV capability during VM feature check", logger.Ctx{"err": err})
+		} else {
+			if string(sev) == "Y\n" {
```

Comment:
Is there a reason you used a separate indented `if` block here rather than make the line above and `else if`?